File: Maths_And_Stats/Statistics/Compound_Interest/Compound_Interest.py
# Python3 program to find compound 
# interest for given values. 
  
def compound_interest(principle, rate, time): 
  
    # Calculates compound interest  
    Amount = principle * (pow((1 + rate / 100), time)) 
    CI = Amount - principle 
    print("Compound interest is", CI) 
  
# Driver Code  
p = float(input("Enter principle amount: "))
r = float(input("Enter rate: "))
t = float(input("Enter time: "))
compound_interest(p, r, t) 

File: Maths_And_Stats/Statistics/Compound_Interest/Compound_Interest.cpp
/*Compound Interest*/

#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main() 
{ 
    double principle, rate, timeSpan;

    cout << "Enter the principle amount" << endl;
    cin >> principle;

    cout << "Enter the compound rate of interest" << endl;
    cin >> rate;
    
    cout << "Enter the time span or number of times the amount gets compounded" << endl;
    cin >> timeSpan;
  
    /* Calculate compound interest */
    /*Formula to calculate final amount after applying compound interest is given by:
	FinalAmount = (principle)*((1 + (rate)/100)^(timeSpan))*/
    double FinalAmount = principle * (pow((1 + rate / 100), timeSpan)); 
  
    cout << "Final amount after applying Compound interest = " << FinalAmount; 
  
    return 0; 
} 